spec/producing/message/changes_payload_generator_spec.rb

```diff
@@ -3,11 +3,13 @@
 require "emque/producing/message/message_with_changeset"
 
 class FakeModel
-  include Virtus.model
+  include Virtus.value_object
 
-  attribute :foo, Integer
-  attribute :bar, Integer
-  attribute :baz, Integer
+  values do
```

Comment:
Do all messages have to be changed to use the `values do` block? that could be something to work on in all of the producers